### DISCUSSION TITLE
ZkSync Bridge

### DISCUSSION THREAD

**Discussion by Nanle100** [Cyfrin/foundry-full-course-cu#2170] [Q&A]
Good day.
I tried browsing into zkSync Bridge site to bridge but my browser keep showing "logging in with metamask"
please what should i do?

**Comment by EngrPips** [comment]
Hello @Nanle100, It seems you do not have a metamask extension downloaded on your browser. If you are using chrome, you can install metamask browser extension from here https://chromewebstore.google.com/detail/metamask/nkbihfbeogaeaoehlefnkodbefgpgknn?hl=en

**Reply by TaiwoEniola** [reply]
im having the same problem too, i have metamask chrome extension but it is still not working

**Reply by EngrPips** [reply]
Hello @TaiwoEniola, Please clear the Chrome browser's browsing cache and then retry. You can use the images below as a guide 

<img width="341" alt="Screenshot 2024-08-19 at 11 04 48" src="https://github.com/user-attachments/assets/e4e62074-295c-4671-9479-db3029e490f3">

<img width="1440" alt="Screenshot 2024-08-19 at 11 05 49" src="https://github.com/user-attachments/assets/5175483b-6a4f-4473-b0f6-862cddd98d24">

<img width="1440" alt="Screenshot 2024-08-19 at 11 06 23" src="https://github.com/user-attachments/assets/7f99790d-a870-4c44-9552-d744369a22a9">

<img width="1440" alt="Screenshot 2024-08-19 at 11 06 56" src="https://github.com/user-attachments/assets/05611995-9106-477f-adde-6eb65718344c">

**Comment by TaiwoEniola** [comment]
Thanks so much for reaching out I am still having the same problem after I
deleted my browsing data.. whenever I select "zksync sepolia testnet" on
the zksync bridge it keeps showing "logging in to Metamask"

**Reply by EngrPips** [reply]
Can you use another browser?

**Reply by TaiwoEniola** [reply]
yes i have tried using another browser still the same

**Reply by EngrPips** [reply]
Why are you discussing the same issue in two separate discussions? Isn't that stressful for you? Try another device let's see if your issue persists.
